src/xhr.js

```diff
@@ -348,6 +382,15 @@ class FetchResponse {
   json() {
     return this.drainText_().then(JSON.parse);
   }
+
+  /**
+   * Reads the xhr response. Could possibly be undefined if server
+   * does not return anything.
+   * @return {?HTMLDocument|undefined}
+   */
+  doc() {
```

Comment:
Is this just [`FetchResponse#text`](https://developer.mozilla.org/en-US/docs/Web/API/Body/text)?